spec/models/repository_spec.rb

```diff
@@ -36,23 +91,41 @@
 
       it { is_expected.to eq('/repo/dummy_repo/') }
     end
-  end
 
-  describe '#remove_repository' do
-    let(:custom_repository) { create :repository, :custom }
-    let(:suse_repository) { create :repository }
+    context 'with no subpath and no trailing slash' do
+      let(:url) { 'http://localhost.com' }
+
+      it { is_expected.to eq '/' }
```

Comment:
also I would use `()`-brackets

Reply:
We have code with and without parentheses all over the place, I wouldn't fret too much about it. If Rubocop complains, then we fix it, otherwise...